File: C PROGRAMMING LANGUAGE/ASSIGNMENTS/ASSIGNMENT-8/alternate.c
int alternate(int[],int);
void main()
{
	int arr[100],size;
	printf("enter size of array: ");
	scanf("%d",&size);
	printf("\nenter array elements: ");
	for(int i=0;i<size;i++)
		scanf("%d",&arr[i]);
	printf("\narray elements: ");
	for(int i=0;i<size;i++)
		printf("%d  ",arr[i]);
	alternate(arr,size);
}
int alternate(int a[],int n)
{
	printf("\nalternate numbers: ");
  for(int i=0;i<n;i+=2)
  	printf("%d  ",a[i]);
}

File: JAVA/ARRAY/Alternate.java
//5.	Print alternate elements in array.

import java.util.Scanner;

class Alternate
{
  public static void main(String[] args)
  {
	Scanner sc = new Scanner(System.in);
	int i=0;
	int[] arr = new int[10];

	System.out.println("enter the array elements: ");
	for(i=0;i<arr.length;i++)
	{
	   arr[i]=sc.nextInt();
 	}

	
	System.out.println("Array elements are: ");
	for(i=0;i<arr.length;i++)
	{
	   System.out.println(arr[i]);
 	}

	System.out.println("alernate Array elements are: ");
	for(i=0;i<arr.length;i+=2)
	{
	   System.out.println(arr[i]);
 	}

    }
}


